src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.lang.reflect.Array;
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        String dash = "____________________________________________________________";
+        String greeting1 = "Pai Kia Bot: Eh harlo! Call me Pai Kia Bot.";
+        String greeting2 = "Pai Kia Bot: What you want?";
+        String greeting3 = "Pai Kia Bot: I will keep a list of your responses ah";
+        String farewell = "Pai Kia Bot: Leave so soon ah? Limpeh sleep first, if got no issue don't disturb me.";
+
+        ArrayList<CheckList> inputList = new ArrayList<>();
+
+        System.out.println(dash);
+        System.out.println(greeting1);
+        System.out.println(greeting2);
+        System.out.println(dash);
+        System.out.print("Your response: ");
+        String input = sc.nextLine();
+        System.out.println("");
+
+        while (!input.equals("bye")) {
+            if (input.equals("list")) {
+                int counter = 1;
+                System.out.println(dash);
+                System.out.println("Pai Kia Bot: i send u your task list ah");
+
+                for (CheckList c : inputList) {
+                    System.out.println(counter + c.toString());
+                    counter++;
+                }
+                System.out.println(dash);
+            } else if (input.length() > 4 && input.substring(0, 5).equals("done ")) {
+                int index = -1;
+                if (!input.equals("done ")) {
+                    index = Integer.parseInt(input.substring(5));
+                }
+                if (index >= 0 && index <= inputList.size()) {
+                    inputList.set(index - 1, inputList.get(index - 1).makeDone());
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: ok i give this task a done chop: " + inputList.get(index - 1).toString().substring(1));
```

Comment:
Maybe you could break this line into separate lines for readability? I noticed lines longer than 100 chars in several other areas too...